util/config/run-in-venv-with-python-bindings.bash

```diff
@@ -7,7 +7,11 @@ CWD=$(cd $(dirname $0) ; pwd)
 # Perform checks and set up environment variables for virtual env.
 source $CWD/run-in-venv-common.bash
 
-if [ ! -f "$chpldeps/chapel.egg-link" ]; then
+python=$($CHPL_HOME/util/config/find-python.sh)
+$python -c 'import chapel' &>/dev/null
+import_exitcode=$?
+
+if [[ ! -f "$chpldeps/chapel.egg-link" || $import_exitcode -ne 0 ]]; then
```

Comment:
Did you run into an issue with the old version of this code?

Reply:
Yes. Even if the bindings where 'installed', python would still be unable to find the bindings. This was also an issue with chplcheck, so not caused by something I did in this script.

Reply:
How did you fix the issue with them being installed? That's odd.

Reply:
I didn't 😞. But this at least prevents the script from being run accidently

Reply:
This was resolved in #23738, and this code has been removed from the PR